app-cdr/dvdisaster/dvdisaster-0.79.5.ebuild

```diff
@@ -20,36 +21,38 @@ done
 unset dvdi_lang
 
 RDEPEND="
-	>=dev-libs/glib-2.32:2
-	nls? ( virtual/libintl )
-	>=x11-libs/gtk+-2.6:2
+	dev-libs/glib:2
 	x11-libs/gdk-pixbuf
-"
-DEPEND="${RDEPEND}
+	x11-libs/gtk+:2
+	nls? ( virtual/libintl )"
+DEPEND="${RDEPEND}"
+BDEPEND="
 	nls? ( sys-devel/gettext )
 	virtual/os-headers
-	virtual/pkgconfig
-"
+	virtual/pkgconfig"
+
+PATCHES=( "${FILESDIR}"/${P}-fno-common.patch )
 
 src_configure() {
-	./configure \
-		--prefix=/usr \
-		--bindir=/usr/bin \
-		--mandir=/usr/share/man \
-		--docdir=/usr/share/doc \
+	econf \
```

Comment:
have you verified that this doesnt spew "unrecognized arguments" warnings because it's some NIH configure script?

Reply:
Yep, it actually has some unrecognized options, changed it back to `./configure` :)